courses/free-courses-ta.md

```diff
@@ -110,6 +110,7 @@ CSE Tamila by Eezytutorials
 * [Python in Tamil for Beginners](https://youtube.com/playlist?list=PLA2UBjeRwle3OLO3qmXTbmCvuTlqhHRVb) - GURUKULA
 * [Python Programming in Tami](https://www.youtube.com/playlist?list=PLWbtDrDnmTHBdEnUKuLNdH2-zKSDD8OA4) - CS in Tamil
 * [Python Tutorial in Tamil](https://youtube.com/playlist?list=PLIFRUdRwOM0_hcLruKbsHWnU5P2uLBgsp) - SANTRA TECHSPOT
+* [#02 Variables and DataTypes Explained 🔋| In Tamil | Python Tutorial Series 🚀| EMC Academy](https://www.youtube.com/watch?v=Rtmgt2Qfqr4) - Error Makes Clever Academy
```

Comment:
We prefer 'playlist' links for YouTube contributions, along with the producer/creator's name. More info in docs/CONTRIBUTING.md#platforms-and-access-notes.

When contributing, use "-" instead of "|" and avoid using emojis.

Reply:
done

Reply:
Remove # 01, use the original title given by the resource, and check free-programming-books-lint for errors.

Reply:
i fixed it, did i do it right?